### ISSUE TITLE
UI: Update Downloads screen

### ISSUE THREAD

**Issue by biliesilva** [HyperPlay-Gaming/hyperplay-desktop-client#95]
As a gamer user, I want to know the status of the game(s) I'm downloading in order to understand and manage their status. 

**Description**
Creating a new design for the Download Manager screen that's simpler and more information rich for currently downloading games. This includes moving information about queued downloads and prior downloads to separate tabs. 

**AC**
* Create three tabs on the Downloads page: 
  - Downloading 
  - Queued 
  - Completed 
* Move queued games' list and status to a new tab - Queued 
* Move completed games' list and status to a new tab - Downloaded 
* Have a progress bar and an estimated time to completion for the currently downloading game 
* Show the following information in table row format about the downloading game: 
  - Icon / Name 
  - Elapsed Time  
  - Download Size 
  - Store 
  - Platform 
  - Pause / Stop Button 
* A graph for download speeds 
* A graph for disc activity

**Comment by wswarren12** [comment]
Only change suggestion I'd have to the download screen would be to show time elapsed rather than "Started At"

**Comment by biliesilva** [comment]
> 

Yeah, definitely better to have this info instead. This one was based on Steam, so they just have the info about "started at".
